packages/flutter_tools/lib/src/isolated/devfs_web.dart

```diff
@@ -95,15 +96,15 @@ class WebExpressionCompiler implements ExpressionCompiler {
     }
 
     return ExpressionCompilationResult(
-        "InternalError: frontend server failed to compile '$expression'",
-        true);
+        "InternalError: frontend server failed to compile '$expression'", true);
```

Comment:
can you revert the formatting docs on this library?

Reply:
done